api/EXT_texture_compression_bptc.json

```diff
@@ -14,7 +14,7 @@
             "version_added": false
           },
           "firefox": {
-            "version_added": "65"
+            "version_added": "68"
```

Comment:
I can't find a Firefox bug for this, but do wonder if "65" might actually be correct, as with a few of the other changes.

Reply:
Performing testing in both Windows and macOS, this extension wasn't present in either Firefox 65 or 67, but was present in both OSes for 68.